### PR TITLE
used controlbar height from skin.json

### PR DESCRIPTION
**Pull request by shruthi17**
This is related to : https://github.com/ooyala/skin-config/pull/145

### CODE REVIEW

**Comment by mistermendez** [comment]
👍

File: config [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 64d81ceeb72ff87e4f941466632af9d5a9847a97
+Subproject commit 2fd9ad1f0ac89789e6c64df477661a399f970d08
```
Comment (mityaha) [inline config]: Why this change?  Did something else has changed in config repository that requires its update here?
